packages/flutter/test/widgets/overscroll_stretch_indicator_test.dart

```diff
@@ -7,11 +7,15 @@
 @Tags(<String>['reduced-test-set'])
 library;
 
+import 'dart:ui' as ui;
+
 import 'package:flutter/rendering.dart';
 import 'package:flutter/widgets.dart';
 import 'package:flutter_test/flutter_test.dart';
 
 void main() {
+  final bool isFragmentShader = ui.ImageFilter.isShaderFilterSupported;
```

Comment:
Explain this more so that maintainers won't be confused.
```suggestion
  // `StretchingOverscrollIndicator` uses a different algorithm when
  // shader is available, therefore the tests must be different depending
  // on shader support.
  final bool isFragmentShader = ui.ImageFilter.isShaderFilterSupported;
```